src/pages/SignUp/SignUp.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
+import './SignUp.scss';
 
 class SignUp extends React.Component {
   render() {
     return (
-      <div>SignUp</div>
-    )
+      <>
+        <form className="signUpForm">
+          <div className="signUpLogo">
+            <img className="logoImage" alt="logo" src="images/mango.png" />
+          </div>
+          <div className="signUpTitle">회원가입</div>
+          <div className="signUpContainer">
+            <input className="inputName" placeholder="이름"></input>
+            <input className="inputEmail" placeholder="이메일"></input>
+            <input className="inputPassword" placeholder="비밀번호"></input>
```

Comment:
```suggestion
            <input type="password" className="inputPassword" placeholder="비밀번호" />
```
- self closing 태그 적용해주세요
- type 속성도 적어주면 좋겠네요!

Reply:
사인업 페이지 로그인 페이지 모두 동일하게 수정하도록 하겠습니다